app/main/posts/views/mode-context-form-filter.directive.js

```diff
@@ -86,6 +90,28 @@ function ModeContextFormFilter($scope, FormEndpoint, PostEndpoint, TagEndpoint,
             }
         });
     }
+    function getPostStats(query) {
+        query = query || PostFilters.getQueryParams(PostFilters.getFilters());
+        var queryParams = _.extend({}, query, {
+            'group_by': 'form',
+            include_unmapped: true
+        });
+        // we want stats for all forms, not just the ones visible right now
```

Comment:
I'd suggest we drop the tags filter too. It seems like it'd get confusing since categories show in the sidebar underneath forms.